whitepaper/chapters/timesync.tex

```diff
@@ -97,7 +97,7 @@ \subsection{Calculation of the Effective Offset}
 This gives the formula for the effective offset $o$
 $$ o = \mathvar{scale} \sum_j I_j \: o_j$$
 
-Note that the influence of an account with large importance is artificially limited because the term $\frac{n}{s} $ caps the scale.
+Note that the influence of an account with large importance is artificially limited because the term $\frac{s}{n} $ caps the scale.
```

Comment:
@Jaguar0625 Please double check if this is correct before merging

Reply:
this is wrong. in the equation `1/(s/n))` == `n/s`